### DISCUSSION TITLE
Does this works on ps3hen with webman installed?

### DISCUSSION THREAD

**Discussion by Silkvine001** [bucanero/ArtemisPS3#27] [Q&A] [answered]
Read the title, i have an non CFW compatible ps3 with HFW hen 3.02 and webman mod installed, also im willing to use artemis to use codes on PvZ GW.

**Comment by stinger101mg** [comment]
Yeah you should be able to use artemis on ps3 hen. I may b mistaken but you should Google installing homebrew pgks on hen, ive only ever owned ps3s that were cfw capable for the cfw specific capabilities vs a hen style exploit every cold boot setup vs a more direct modification of the os itself vs only the current memory image of the software as it were in any given moment after a cold boot. So having never used ps3hen I can't say with 100% certainty so please google s your best friend and mind the ram limits of the ps3 itself if you've got too much in use and not enough free for the amount of cheats currently enabled and the corresponding memory requirements to make alterations to in use ram directly related to the amount of cheats enabled. So with webman installed it uses a background plugin as a base for its software which exists in ram as does the rest of the software currently in use, which limits the amount of free ram you have available for apps games and most of all your plugins because artemis uses the same plugin mechanism and the same ram banks everything else operates on. What I'm getting at is if you have webman enabled fully you may not have enough ram free for artemis and a game running all at once, I experienced this personally on some fw with webman fully enabled, the issue arose after selecting cheats and loading the game successful artemis attachment, as the game would save your memory consumption goes up and you could run out of ram causing a system lockup and hard reboot. To avoid this note your ram use and see what you can do about limiting webman usage to maximize your available ram.

**Comment by bucanero** [comment, marked as answer]
Yes, Artemis works on HEN (even with webman), but as @stinger101mg said you need to keep an eye on the memory usage. 

If you're having issues such as game crashing after applying codes, then try to unload webman to free some memory when you use Artemis.

**Reply by DawTheD** [reply]
Thanks for the tip, that might be the problem why i am on black screen after setting mod to r5
